### ISSUE TITLE
Draw Subgroups

### ISSUE THREAD

**Issue by jeanbellynck** [jeanbellynck/Cayley-Graph-Generator#48]
We add a secondary Generator where you can enter the generating set of a subgroup. The subgroup is then drawn using arrows inside the ambient graph.

**Comment by p4jo** [comment]
We could use gap to show a better presentation for this

**Comment by jeanbellynck** [comment]
> We could use GAP to show a better presentation for these or to enumerate these or to enumerate cosets etc.
> Maybe learn and educate about algorithms? Todd-Coxeter etc

Definitely! If we want to educate we will need some kind of tooltips or information buttons.

**Comment by jeanbellynck** [comment]
I also recently learned about the Reidemeister-Schreier Theorems for subgroups. Those might be interesting too
